Kitodo/src/main/java/org/kitodo/production/forms/dataeditor/StructurePanel.java

```diff
@@ -538,12 +538,14 @@ public void show() {
 
     private void restoreSelection(String rowKey, TreeNode parentNode) {
         for (TreeNode childNode : parentNode.getChildren()) {
```

Comment:
Can `parentNode` have children which are Null?

Reply:
Potentially, yes, because `children` is just a list of `DefaultTreeNode`,  itself an implementation of the interface `TreeNode`, and instances of every non-primitive type in Java can be `null`. Whether this can actually be the case in the Kitodo context is a different story, but shouldn't be the deciding factor for adding null checks, IMHO.